diff --git a/node_modules/@rhds/elements/react/elements/rh-button/rh-button.d.ts b/node_modules/@rhds/elements/react/elements/rh-button/rh-button.d.ts
index 9a69346..0b9e203 100644
--- a/node_modules/@rhds/elements/react/elements/rh-button/rh-button.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-button/rh-button.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-button/rh-button.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhButton } from '@rhds/elements/elements/rh-button/rh-button.js';
+import type { RhButton } from '@rhds/elements/rh-button/rh-button.js';
 export const Button: ReactWebComponent<RhButton, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-button/rh-button.js b/node_modules/@rhds/elements/react/elements/rh-button/rh-button.js
index 25dd4d8..7fa8e34 100644
--- a/node_modules/@rhds/elements/react/elements/rh-button/rh-button.js
+++ b/node_modules/@rhds/elements/react/elements/rh-button/rh-button.js
@@ -1,7 +1,7 @@
 // elements/rh-button/rh-button.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhButton as elementClass } from '@rhds/elements/elements/rh-button/rh-button.js';
+import { RhButton as elementClass } from '@rhds/elements/rh-button/rh-button.js';
 export const Button = createComponent({
   tagName: 'rh-button',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.d.ts b/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.d.ts
index 25ceebd..ae03168 100644
--- a/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-cta/rh-cta.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhCta } from '@rhds/elements/elements/rh-cta/rh-cta.js';
+import type { RhCta } from '@rhds/elements/rh-cta/rh-cta.js';
 export const Cta: ReactWebComponent<RhCta, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.js b/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.js
index 901dc8a..220a6bf 100644
--- a/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.js
+++ b/node_modules/@rhds/elements/react/elements/rh-cta/rh-cta.js
@@ -1,7 +1,7 @@
 // elements/rh-cta/rh-cta.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhCta as elementClass } from '@rhds/elements/elements/rh-cta/rh-cta.js';
+import { RhCta as elementClass } from '@rhds/elements/rh-cta/rh-cta.js';
 export const Cta = createComponent({
   tagName: 'rh-cta',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.d.ts b/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.d.ts
index 0d90a8f..4dd7dcc 100644
--- a/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-dialog/rh-dialog.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhDialog } from '@rhds/elements/elements/rh-dialog/rh-dialog.js';
+import type { RhDialog } from '@rhds/elements/rh-dialog/rh-dialog.js';
 export const Dialog: ReactWebComponent<RhDialog, { onOpen: 'open'; onClose: 'close'; onCancel: 'cancel' }>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.js b/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.js
index 2930651..948cd31 100644
--- a/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.js
+++ b/node_modules/@rhds/elements/react/elements/rh-dialog/rh-dialog.js
@@ -1,7 +1,7 @@
 // elements/rh-dialog/rh-dialog.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhDialog as elementClass } from '@rhds/elements/elements/rh-dialog/rh-dialog.js';
+import { RhDialog as elementClass } from '@rhds/elements/rh-dialog/rh-dialog.js';
 export const Dialog = createComponent({
   tagName: 'rh-dialog',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.d.ts
index bb44c51..3776162 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer-block.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooterBlock } from '@rhds/elements/elements/rh-footer/rh-footer-block.js';
+import type { RhFooterBlock } from '@rhds/elements/rh-footer/rh-footer-block.js';
 export const FooterBlock: ReactWebComponent<RhFooterBlock, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.js
index 5ce9c71..6d6ca02 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-block.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer-block.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooterBlock as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer-block.js';
+import { RhFooterBlock as elementClass } from '@rhds/elements/rh-footer/rh-footer-block.js';
 export const FooterBlock = createComponent({
   tagName: 'rh-footer-block',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.d.ts
index 6a9efe4..40b1429 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer-copyright.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooterCopyright } from '@rhds/elements/elements/rh-footer/rh-footer-copyright.js';
+import type { RhFooterCopyright } from '@rhds/elements/rh-footer/rh-footer-copyright.js';
 export const FooterCopyright: ReactWebComponent<RhFooterCopyright, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.js
index fff4b16..d2fb420 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-copyright.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer-copyright.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooterCopyright as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer-copyright.js';
+import { RhFooterCopyright as elementClass } from '@rhds/elements/rh-footer/rh-footer-copyright.js';
 export const FooterCopyright = createComponent({
   tagName: 'rh-footer-copyright',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.d.ts
index 5b17160..6f66a53 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer-links.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooterLinks } from '@rhds/elements/elements/rh-footer/rh-footer-links.js';
+import type { RhFooterLinks } from '@rhds/elements/rh-footer/rh-footer-links.js';
 export const FooterLinks: ReactWebComponent<RhFooterLinks, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.js
index a7c5519..b9ab18e 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-links.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer-links.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooterLinks as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer-links.js';
+import { RhFooterLinks as elementClass } from '@rhds/elements/rh-footer/rh-footer-links.js';
 export const FooterLinks = createComponent({
   tagName: 'rh-footer-links',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.d.ts
index 3ee3c75..77dd07f 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer-social-link.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooterSocialLink } from '@rhds/elements/elements/rh-footer/rh-footer-social-link.js';
+import type { RhFooterSocialLink } from '@rhds/elements/rh-footer/rh-footer-social-link.js';
 export const FooterSocialLink: ReactWebComponent<RhFooterSocialLink, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.js
index e88b3ab..b50f782 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-social-link.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer-social-link.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooterSocialLink as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer-social-link.js';
+import { RhFooterSocialLink as elementClass } from '@rhds/elements/rh-footer/rh-footer-social-link.js';
 export const FooterSocialLink = createComponent({
   tagName: 'rh-footer-social-link',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.d.ts
index 094141b..2d3f0b8 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer-universal.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooterUniversal } from '@rhds/elements/elements/rh-footer/rh-footer-universal.js';
+import type { RhFooterUniversal } from '@rhds/elements/rh-footer/rh-footer-universal.js';
 export const FooterUniversal: ReactWebComponent<RhFooterUniversal, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.js
index 01dbdf0..633a526 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer-universal.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer-universal.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooterUniversal as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer-universal.js';
+import { RhFooterUniversal as elementClass } from '@rhds/elements/rh-footer/rh-footer-universal.js';
 export const FooterUniversal = createComponent({
   tagName: 'rh-footer-universal',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.d.ts
index 20535e5..3fae4ee 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-footer.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhFooter } from '@rhds/elements/elements/rh-footer/rh-footer.js';
+import type { RhFooter } from '@rhds/elements/rh-footer/rh-footer.js';
 export const Footer: ReactWebComponent<RhFooter, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.js
index a33d163..dc7a674 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-footer.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-footer.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhFooter as elementClass } from '@rhds/elements/elements/rh-footer/rh-footer.js';
+import { RhFooter as elementClass } from '@rhds/elements/rh-footer/rh-footer.js';
 export const Footer = createComponent({
   tagName: 'rh-footer',
   elementClass,
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.d.ts b/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.d.ts
index 9a541d4..fc631d1 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.d.ts
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.d.ts
@@ -1,5 +1,5 @@
 // elements/rh-footer/rh-global-footer.js
 import type { ReactWebComponent } from '@lit/react';
-import type { RhGlobalFooter } from '@rhds/elements/elements/rh-footer/rh-global-footer.js';
+import type { RhGlobalFooter } from '@rhds/elements/rh-footer/rh-global-footer.js';
 export const GlobalFooter: ReactWebComponent<RhGlobalFooter, {}>;
   
diff --git a/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.js b/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.js
index 4f87243..844d2ba 100644
--- a/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.js
+++ b/node_modules/@rhds/elements/react/elements/rh-footer/rh-global-footer.js
@@ -1,7 +1,7 @@
 // elements/rh-footer/rh-global-footer.js
 import { createComponent } from '@lit/react';
 import react from 'react';
-import { RhGlobalFooter as elementClass } from '@rhds/elements/elements/rh-footer/rh-global-footer.js';
+import { RhGlobalFooter as elementClass } from '@rhds/elements/rh-footer/rh-global-footer.js';
 export const GlobalFooter = createComponent({
   tagName: 'rh-global-footer',
   elementClass,
